KamerConnect.View.MAUI/Views/UpdateAccountsForm.xaml.cs

```diff
@@ -54,4 +67,19 @@ private string GetContentType(string fileName)
             _ => "application/octet-stream"
         };
     }
+    
+    public string LoadLocalEnv()
```

Comment:
Er is een functie die aan het beginnen van het runnen van de code die al uitgevoerd word die de env laad. Met Environment.GetEnvironmentVariable("{Variabele naam}") kan je alle variabele al ophalen.

Reply:
gefixed